app/Http/Livewire/CategoryComponent.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\Http\Livewire;
+
+use App\Models\Category;
+use App\Models\Images;
+use App\Models\Product;
+use Livewire\Component;
+use Livewire\WithPagination;
+
+class CategoryComponent extends Component
+{
+    public $sorting;
+    public $pagesize;
+    public $category_slug;
+
+    public function mount($category_slug)
+    {
+        $this->sorting = config('constant.default_sorting');
+        $this->pagesize = config('constant.default_pagesize');
+        $this->category_slug = $category_slug;
+    }
+
+    use WithPagination;
+    public function render()
+    {
+        $category = Category::where('slug', $this->category_slug)->first();
+        $category_id = $category->id;
+        $category_name = $category->name;
```

Comment:
```suggestion
        $categoryId = $category->id;
        $categoryName = $category->name;
```
chú ý cách đặt tên biến